volttron/platform/auth.py

```diff
@@ -139,7 +139,7 @@ def read_auth_file(self):
                 _log.debug("Sending auth updates to peers")
                 # Give it few seconds for platform to startup or for the
                 # router to detect agent install/remove action
-                gevent.sleep(5)
+                gevent.sleep(2)
                 self._send_update()
```

Comment:
Is there any way to detect whether this sleep is necessary or not?

Reply:
No. I can't of think of any way. I can remove it if it is a nuisance.